sourcetracker/_sourcetracker.py

```diff
@@ -975,3 +987,15 @@ def collate_gibbs_results(all_envcounts, all_env_assignments,
             fts = None
 
     return props, props_stds, fts
+
+
+def plot_mpm(mpm, cm=plt.cm.viridis):
+    '''Make a basic mixing proportion histogram.'''
+    fig = plt.figure()
+    ax = fig.add_subplot(1, 1, 1)
+    sns.heatmap(mpm, vmin=0, vmax=1.0, cmap=cm, annot=True, linewidths=.5,
+                ax=ax)
+    ax.set_xlabel('Sources')
```

Comment:
Could you make these labels optional parameters (where these values are the defaults)? You don't need to hook that up to the CLI or anything, it just might be handy for the API (e.g., I often don't want labels, because I mess with the plots in Illustrator before paper submission).